docs/src/test-reporters-js.md

```diff
@@ -429,6 +429,8 @@ npx playwright test --reporter="./myreporter/my-awesome-reporter.ts"
 * [Currents](https://www.npmjs.com/package/@currents/playwright)
 * [GitHub Actions Reporter](https://www.npmjs.com/package/@estruyf/github-actions-reporter)
 * [GitHub Pull Request Comment](https://github.com/marketplace/actions/playwright-report-comment)
+* [Mail Reporter](https://www.npmjs.com/package/playwright-mail-reporter)
+* [Microsoft Teams Reporter](https://www.npmjs.com/package/playwright-msteams-reporter)
```

Comment:
Are you interested in moving the Teams reporter under the [Playwright Community Organisation](https://github.com/playwright-community) while maintaining the ownership?

Reply:
Sound fine to me

Reply:
Status: Transferred the repository over to the Playwright Community, ideally we publish now also under @playwright-community under NPM. This name is not available right now, so I reached out to them for assistance.